subsys/bluetooth/services/fast_pair/fp_gatt_service.c

```diff
@@ -57,11 +62,32 @@ enum fp_msg_type {
 	FP_MSG_ACTION_REQ               = 0x10,
 };
 
-struct msg_key_based_pairing_req {
+enum additional_action {
+	ADDITIONAL_ACTION_NOTIFY_PN_BIT_POS,
+	ADDITIONAL_ACTION_COUNT,
+};
+
+struct msg_kbp_req {
+	uint8_t seeker_address[BT_ADDR_SIZE];
+};
+
+struct msg_action_req {
```

Comment:
Maybe name it `msg_action_req_data` (that's not complete message but just additional message data)